streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapterTest.java

```diff
@@ -140,17 +139,15 @@ private void verifyDBOptionsMethodCall(final Method method) throws Exception {
         final Object[] parameters = getDBOptionsParameters(method.getParameterTypes());
 
         try {
-            reset(mockedDbOptions);
-            replay(mockedDbOptions);
             method.invoke(optionsFacadeDbOptions, parameters);
-            verify();
-            fail("Should have called DBOptions." + method.getName() + "()");
+            final Collection<Invocation> invocations = mockingDetails(mockedDbOptions).getInvocations();
+            final Set<String> invokedMethodNames = invocations.stream().map(invocation -> invocation.getMethod().getName()).collect(Collectors.toSet());
+            assertTrue(invokedMethodNames.contains(method.getName()));
```

Comment:
please add a line which will be printed when this assertion fails..."Should have called DBOptions." + method.getName() + "()"